rabbitmq/manifest.json

```diff
@@ -11,7 +11,7 @@
     "mac_os",
     "windows"
   ],
-  "version": "1.3.1",
+  "version": "1.3.2",
```

Comment:
Given that this would actually be a small new "feature" it would technically be `1.4.0` despite the small change.